### PR TITLE
US114854 - US114855 - Get BSI using the latest facet-filter-sort

### PR DESCRIPTION
**Pull request by svanherk**
Bump `d2l-facet-filter-sort`, `d2l-common`, `d2l-activities` and `d2l-my-courses`

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1209,9 +1209,9 @@
       }
     },
     "@brightspace-ui/core": {
-      "version": "1.66.0",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui/core/-/core-1.66.0.tgz",
-      "integrity": "sha512-+k/2KUctFPDwgnrGngMjRX/hLu73SO82P//bAIIkfPTtpVgG3VtxK1yPpc5F6WAdIVL/VVCXB8vLuOOcTDLw+Q==",
+      "version": "1.68.0",
```
Comment (svanherk) [inline package-lock.json:1212]: FYI @dlockhart @dbatiste @AllanKerr @margaree @m6jones @zinabat

File: package-lock.json [hunk 2]
```diff
@@ -1226,9 +1226,9 @@
       }
     },
     "@brightspace-ui/intl": {
-      "version": "3.0.9",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui/intl/-/intl-3.0.9.tgz",
-      "integrity": "sha512-fvVkfLKDdff1IrQPBnE6GyPgSS+IQ/n0lH/0Fp9/qwi1NqtQNVikXaFNnSc4M7vmLUBmQzh/VBzVWxDLjOYnEA==",
+      "version": "3.0.10",
```
Comment (svanherk) [inline package-lock.json:1229]: FYI @margaree

File: package-lock.json [hunk 3]
```diff
@@ -4287,8 +4297,8 @@
         "array-find-index": "^1.0.1"
       }
     },
-	  "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#e95616d07e50f5572fd9e48cc499ac8de4352a03",
+    "d2l-activities": {
```
Comment (svanherk) [inline package-lock.json:4300]: FYI @imcdonald-d2l

File: package-lock.json [hunk 4]
```diff
@@ -4417,7 +4427,7 @@
       }
     },
     "d2l-awards-leaderboard-ui": {
-      "version": "github:Brightspace/awards-leaderboard-ui#6c70913bd3e31d03afbfe6d235f9fb39debd74d1",
+      "version": "github:Brightspace/awards-leaderboard-ui#182158406ad37d0d4bde92df02c9d9bcd27044e7",
```
Comment (svanherk) [inline package-lock.json:4430]: FYI @cbelan-d2l

File: package-lock.json [hunk 5]
```diff
@@ -4557,7 +4567,7 @@
       }
     },
     "d2l-cpd": {
-      "version": "github:Brightspace/continuous-professional-development#50ce1d8e058c40fe48f92d3b99ac6dbf493cd4da",
+      "version": "github:Brightspace/continuous-professional-development#7d99e2cfcfbff221fbd7b8026c2b7c50e13f713a",
```
Comment (svanherk) [inline package-lock.json:4570]: FYI @AllanKerr

File: package-lock.json [hunk 6]
```diff
@@ -4907,7 +4915,7 @@
       }
     },
     "d2l-my-dashboards": {
-      "version": "github:Brightspace/d2l-my-dashboards-ui#41468de5f99a02d6a1ab4e82fbdb38004a074cb3",
+      "version": "github:Brightspace/d2l-my-dashboards-ui#fae40b3c5936c34b078e75fcf62dff196bd0ada4",
```
Comment (svanherk) [inline package-lock.json:4918]: FYi @AllanKerr @cbellini

File: package-lock.json [hunk 7]
```diff
@@ -4977,7 +4985,7 @@
       "dev": true
     },
     "d2l-organizations": {
-      "version": "github:BrightspaceHypermediaComponents/organizations#5203c9669df1b17ef69c1c27bfc8b6ec68daa987",
+      "version": "github:BrightspaceHypermediaComponents/organizations#7df332b9e8e30398ce2fcefc3ed3ed27c1913099",
```
Comment (svanherk) [inline package-lock.json:4988]: FYI @AllanKerr

File: package-lock.json [hunk 8]
```diff
@@ -5110,7 +5118,7 @@
       }
     },
     "d2l-rubric": {
-      "version": "github:Brightspace/d2l-rubric#7c6eecbae48ce80ba60ce0aa066cf8a93dbe4713",
+      "version": "github:Brightspace/d2l-rubric#a2c53342d8b7d06f3a444581aac2c81d6fb09f45",
```
Comment (svanherk) [inline package-lock.json:5121]: FYI @djwester @dlockhart

File: package-lock.json [hunk 9]
```diff
@@ -5248,17 +5256,8 @@
...
-      "from": "github:BrightspaceUI/tabs#semver:^1",
-      "dev": true,
-      "requires": {
-        "@brightspace-ui/core": "^1.40.0",
-        "@polymer/polymer": "^3.0.0"
-      }
-    },
     "d2l-teacher-course-creation": {
-      "version": "github:Brightspace/custom-teacher-course-creation#3d74de3f6bf1bc82331a532ed096d912c9c4a828",
+      "version": "github:Brightspace/custom-teacher-course-creation#4c2d5342cdfc1a8bc1d42a0ec51baccc379e94ea",
```
Comment (svanherk) [inline package-lock.json:5260]: FYi @AllanKerr

File: package-lock.json [hunk 10]
```diff
@@ -5248,17 +5256,8 @@
         "stickyfilljs": "^2.1.0"
       }
     },
-    "d2l-tabs": {
```
Comment (svanherk) [inline package-lock.json:5251]: 🎉

File: package-lock.json [hunk 11]
```diff
@@ -4688,20 +4688,17 @@
...
       "dev": true,
       "requires": {
+        "@brightspace-ui-labs/multi-select": "github:BrightspaceUILabs/multi-select#semver:^3",
+        "@brightspace-ui/core": "^1.43.3",
         "@polymer/polymer": "^3.0.0",
-        "d2l-button": "github:BrightspaceUI/button#semver:^5",
-        "d2l-colors": "github:BrightspaceUI/colors#semver:^4",
-        "d2l-dropdown": "github:BrightspaceUI/dropdown#semver:^7",
-        "d2l-icons": "github:BrightspaceUI/icons#semver:^6",
-        "d2l-inputs": "github:BrightspaceUI/inputs#semver:^2",
```
Comment (bemailloux) [inline package-lock.json:4700]: :D :D :D

File: package-lock.json (hunk 10 above)
Comment (dlockhart) [reply package-lock.json:5251]: Nice!

**Pull request merged**